flowey/flowey_lib_common/src/_util/extract.rs

```diff
@@ -125,7 +135,7 @@ impl ClaimVar for ExtractTarBz2Deps {
 pub fn extract_tar_bz2_if_new_deps(ctx: &mut NodeCtx<'_>) -> ExtractTarBz2Deps {
     ExtractTarBz2Deps {
         persistent_dir: ctx.persistent_dir(),
-        lbzip2_installed: ctx.reqv(|v| crate::install_apt_pkg::Request::Install {
+        lbzip2_installed: ctx.reqv(|v| crate::install_dist_pkg::Request::Install {
             package_names: vec!["lbzip2".into()],
```

Comment:
this doesn't need updating?

Reply:
Nope, the package name is the same on Fedora:
```console
$ sudo dnf install lbzip2
[sudo] password for krom:
Fedora 40 - x86_64                                                                       70 kB/s |  29 kB     00:00
Fedora 40 - x86_64 - Updates                                                             77 kB/s |  27 kB     00:00
Fedora 40 - x86_64 - Updates                                                            2.1 MB/s | 4.3 MB     00:02
Last metadata expiration check: 0:00:01 ago on Mon Oct 21 12:39:18 2024.
Package lbzip2-2.5-29.20171011gitb6dc48a.fc40.x86_64 is already installed.
Dependencies resolved.
Nothing to do.
Complete!
```